participants/Decentral Myvlog.md

```diff
@@ -0,0 +1,59 @@
+
+(0) Project Name and Team Slack Handles
+project's name:Decentral Myvlog
+Filecoin Slack: myvlog@decentral.cc

```

Comment:
Is `@Decentral Tech` your username on slack?

Reply:
Hi, @cwaring , the name on slack is @Decentral Myvlog, same to the project name, Thanks~~